heroku-22/setup.sh

```diff
@@ -73,6 +73,7 @@ packages=(
   libgd3
   libgdk-pixbuf2.0-0
   libgdk-pixbuf2.0-common
+  libgeoip1
```

Comment:
For Heroku-24 I removed `libgeoip-dev`, `libgeoip1`, `geoip-database` etc for the reason that they were (a) broken on older stacks, (b) clearly not used by anyone, since no one had reported the breakage.

Given they aren't supported on Heroku-24, should we skip adding them back to older stacks? If anyone actually does start depending on those packages, they are only going to then have to find another solution (eg Apt buildpack) regardless when they upgrade to Heroku-24 otherwise.

(This package is very small, so this isn't about the size, more us being clear about intent and easing migration to newer stacks etc)

Reply:
Sounds reasonable.